validation/td-validation.ttl

```diff
@@ -1843,7 +1843,11 @@
           <code>format</code> value to perform additional
           validation accordingly.</span> <span class="rfc2119-assertion" id="td-format-validation-other-values">When a value that is
           not found in the known set of values is assigned to
-          <code>format</code>, such a validation <em class="rfc2119" title="SHOULD">SHOULD</em> succeed.</span></p><p class="note">The <code>format</code> term is not widely implemented by JSON Schema tools. 
+          <code>format</code>, such a validation <em class="rfc2119" title="SHOULD">SHOULD</em> succeed.</span></p>
+          <span class="rfc2119-assertion" id="td-dataSchema-anyType">
+          Vocabulary terms that are typed as <code>any</code> <code>type</code> (e.g., <code>const</code>, <code>default</code>) <em class="rfc2119" title="MUST">MUST</em> follow data types compatible with JSON Schema (boolean, integer, number, string, object, array, or null).
```

Comment:
I think that we do not need another assertion for this, just explaining. Also, any type is not necessarily about JSON Schema, it simply means a json value that can be of any shape. This explanation is a bit too complicated in my opinion.